src/renderer/api-decorator.js

```diff
@@ -178,9 +178,8 @@ limitations under the License.
                 e.preventDefault();
 
                 const identity = entityInfo.entityType === 'iframe' ? entityInfo.parent : entityInfo;
-                const options = getWindowOptionsSync(identity);
 
-                if (options.contextMenu) {
+                if (initialOptions.contextMenu) {
```

Comment:
This is going to break if someone updates the `contextMenu` option after window spin up

Reply:
good catch, changing this to query the browser for state.